project/Dependencies.scala

```diff
@@ -41,10 +33,5 @@ object Dependencies {
     commonsLang, // for levenshtein distance impl
     akkaRemoting % Provided,
```

Comment:
I think this can be changed to `% Test`

Reply:
tried but won't work

Reply:
It needed to bring `akka-actor`. Otherwise is not even pulling `akka.actor`
```
[error] /Users/francisco/akka-diagnostics/akka-diagnostics/src/main/scala/akka/diagnostics/ConfigChecker.scala:10:13: object actor is not a member of package akka
[error] import akka.actor.{ ActorSystem, ExtendedActorSystem }
```

Reply:
akka-actor must still be included, I have changed in https://github.com/akka/akka-diagnostics/pull/25